reference/6/Microsoft.PowerShell.Core/New-PSSessionConfigurationFile.md

```diff
@@ -15,7 +15,7 @@ Creates a file that defines a session configuration.
 
 ## SYNTAX
 
-```
+```powershell
```

Comment:
The Syntax blocks should not have a 'powershell' language tag on them because they are not executable like the examples. See https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/contributing/5-FORMATTING-CODE.md

Reply:
Got it, thank you!

Reply:
@sdwheeler it looks like I was able to work around Platy PS boundaries. Please kindly have a look.